vendor.conf

```diff
@@ -125,7 +125,7 @@ github.com/containerd/ttrpc 94dde388801693c54f88a6596f713b51a8b30b2d
 github.com/gogo/googleapis 08a7655d27152912db7aaf4f983275eaf8d128ef
 
 # cluster
-github.com/docker/swarmkit 8852e8840e30d69db0b39a4a3d6447362e17c64f
+github.com/docker/swarmkit 324a450ca695fae75b7bd00d6b83debddea18462
```

Comment:
I would suggest updating to the latest which is cfa742c8abe6f8e922f6e4e920153c408e7d9c3b @selansen